tests/cli/commands/test_kubernetes_command.py

```diff
@@ -40,3 +43,90 @@ def test_generate_dag_yaml(self):
             self.assertEqual(len(os.listdir(out_dir)), 6)
             self.assertTrue(os.path.isfile(out_dir + file_name))
             self.assertGreater(os.stat(out_dir + file_name).st_size, 0)
+
+    @mock.patch('kubernetes.client.CoreV1Api.delete_namespaced_pod')
```

Comment:
Can you move it to new test class? I try to follow the convention - one class = one command to make it easier to find the appropriate tests

Reply:
Done in https://github.com/apache/airflow/pull/11802/commits/5e4a0785ea10d2cb9295423959e53f91b8a5469d